encryption_works.md

```diff
@@ -339,53 +343,54 @@ Snowden was right to be cautious and to insist that he check Gellman's new PGP f
 
-Because Tails is not practical for daily computer use, it's a good idea to get into the habit of using OTR and PGP in your normal operating system as well. Tails won't help blunt the effects of dragnet surveillance by itself, but encrypting as much as we can on a daily basis will.
+Since Tails isn't a practical choice for daily computer use, setting up Tor, PGP, and OTR encryption on your main operating system is also a good idea. 
 
-Every time you boot Tails you start from a clean slate. Anything you did in your previous session on Tails gets erased and the system is reverted back to the default state. This means that even if you get infected with malware while using Tails, the next time you boot into it the malware will be gone.
+Every time you boot Tails you start from a clean state. Anything and everything you did previously gets erased and the system reverts back to its default out-of-the-box state. It's designed this way so that if you get infected with malware while using Tails, it's gone the next time you boot up.
 
-You can get started using Tails by [downloading](https://tails.boum.org/download/index.en.html) the DVD image and burning it to a DVD. You then need to boot to this DVD. This step is different depending on what model computer you have, but it often involves entering your BIOS and changing your boot order so your computer tries booting from DVD before it tries your hard drive. On newer PCs you might need to disable [UEFI "secure boot"](https://en.wikipedia.org/wiki/Unified_Extensible_Firmware_Interface#Booting) in the BIOS as well, which is the crypto that's used to make sure your computer will only boot to digitally signed versions of Windows (which, in affect, makes it harder for people to boot into non-Windows operating systems). The [Tails website](https://tails.boum.org/download/index.en.html#start) has more information on booting Tools from a DVD or USB stick.
+It's a good idea to install Tails on a fresh storage device (either a DVD or a USB stick) to ensure everything works as it should. There are two ways of installing Tails:
 
-After booting to the DVD you have the option to install Tails on a USB stick, which is especially useful because it allows you to configure a [persistent volume](https://tails.boum.org/doc/first_steps/persistence/index.en.html), an encrypted section of your USB stick to store your data. Despite starting from a clean slate each time you boot up, it's important for you to be able to have access to your OTR and PGP keys, your Claws Mail (more below) and Pidgin settings, and any documents you're working with. Your persistent volume allows you to do this.
+- 1) Using the command line and the Tails .iso file to create a Tails instance on a removable storage device.
+- 2) Using an existing Tails installation and cloning the operating system onto a second storage device.
```

Comment:
I think you should move the Tails persistence section here, since using persistence requires making specific choices about your method of installation and it'd be good to learn that sooner rather than later.